### DISCUSSION TITLE
Làm thế nào để chạy repo.

### DISCUSSION THREAD

**Discussion by hainv01** [haihy2001/mobile#29] [Q&A] [answered]
Làm thế nào để chạy repo.

**Comment by namvt48** [comment, marked as answer]
Clone the repo
git clone https://github.com/hainv01/mobile.git
Run the app
flutter run

**Reply by datntrong** [reply]
=))))

**Comment by datntrong** [comment]
Tự làm đi, hỏi nhiều
